lib/factory_bot_rails/railtie.rb

```diff
@@ -22,6 +22,10 @@ class Railtie < Rails::Railtie
       Reloader.new(app, config).run
     end
 
+    config.after_initialize do
+      FactoryBot.reload
```

Comment:
The previous version here was doing:
```ruby 
FactoryBot.find_definitions
```
This new version by doing reload also does:
```ruby
Internal.reset_configuration
register_default_strategies
register_default_callbacks
find_definitions
```
Maybe this question is completely out of context but do we need to re-register all the callbacks and strategies upon initialization? Aren't we doing that somewhere else already?

Reply:
Good question. The strategies and callbacks are stored on the configuration object, so `Internal.reset_configuration` wipes them out and we have to re-register them.